src/test/integration/IntegrationBase.t.sol

```diff
@@ -351,6 +347,234 @@ abstract contract IntegrationBase is IntegrationDeployer {
                        TIME TRAVELERS ONLY BEYOND THIS POINT
     *******************************************************************************/
 
+    /*******************************************************************************
+                         SNAPSHOT ASSERTIONS: ALLOCATIONS
+    *******************************************************************************/
+
+    function assert_Snap_Allocations_Modified(
+        User operator,
+        AllocateParams memory allocateParams,
+        bool completed,
+        string memory err
+    ) internal {
+        Allocation[] memory curAllocs = _getAllocations(operator, allocateParams.operatorSet, allocateParams.strategies);
+        Allocation[] memory prevAllocs = _getPrevAllocations(operator, allocateParams.operatorSet, allocateParams.strategies);
+        Magnitudes[] memory curMagnitudes = _getMagnitudes(operator, allocateParams.strategies);
+        Magnitudes[] memory prevMagnitudes = _getPrevMagnitudes(operator, allocateParams.strategies);
+
+        for (uint i = 0; i < allocateParams.strategies.length; i++) {
+            Allocation memory curAlloc = curAllocs[i];
+            Allocation memory prevAlloc = prevAllocs[i]; 
+            
+            // Check allocations
+            if (completed) {
+                assertEq(curAlloc.currentMagnitude, allocateParams.newMagnitudes[i], string.concat(err, " (currentMagnitude)"));
+                assertEq(curAlloc.pendingDiff, 0, string.concat(err, " (pendingDiff)"));
+                assertEq(curAlloc.effectBlock, 0, string.concat(err, " (effectBlock)"));
+            } else {
+                assertEq(
+                    curAlloc.currentMagnitude, 
+                    prevAlloc.currentMagnitude, 
+                    string.concat(err, " (currentMagnitude)")
+                );
+                assertEq(
+                    curAlloc.pendingDiff, 
+                    prevAlloc.pendingDiff + int128(int64(allocateParams.newMagnitudes[i])), 
+                    string.concat(err, " (pendingDiff)")
+                );
+
+                (, uint32 delay) = allocationManager.getAllocationDelay(address(operator));
+
+                assertEq(
+                    curAlloc.effectBlock, 
+                    block.number + delay, 
+                    string.concat(err, " (effectBlock)")
+                );
+            }
+
+            // Check magnitudes
+            Magnitudes memory curMagnitude = curMagnitudes[i];
+            Magnitudes memory prevMagnitude = prevMagnitudes[i];
+
+            // if (isAllocation) ...
+            if (prevAlloc.currentMagnitude < allocateParams.newMagnitudes[i]) {
+                uint256 delta = allocateParams.newMagnitudes[i] - prevAlloc.currentMagnitude;
+                
+                if (completed) {
+                    // FIXME: fix completed case
```

Comment:
If its an allocation, doesn't it add to encumbered magnitude(and therefore reducing allocatable by same amount)
regardless if the delay has passed? So we can remove the if check and the encumbered/allocatable values are always updated regardless if completed or not
https://github.com/Layr-Labs/eigenlayer-contracts/blob/d3109212d8869ebba791790bfe0e22bdfadd7e5f/src/contracts/core/AllocationManager.sol#L206-L211

Reply:
fixed in [d7a5d61](https://github.com/Layr-Labs/eigenlayer-contracts/pull/934/commits/d7a5d61437737a470dd4d22220e8a818d21c2ac1)